spell/psm-checklists.md

```diff
@@ -0,0 +1,113 @@
+        - [ ] Contract has been audited by an external party
+            - [ ] Audit reports are public
+            - [ ] All issues found were fixed or acknowledged
+            - [ ] No code changes are done after final Audit report
+        - [ ] Deployed contract matches the code in the [repo](https://github.com/makerdao/dss-lite-psm/)
+        - [ ] Deployer no longer has ward privileged access
+        - [ ] `MCD_PAUSE_PROXY` has been authed (i.e. `require(WardsLike(LITE_PSM).wards(MCD_PAUSE_PROXY) == 1)`)
+        - [ ] `DssLitePsm` has `type(uint256).max` approval to spend `gem` on behalf of `pocket` (i.e. `require(GemLike(GEM).allowance(POCKET, LITE_PSM) == type(uint256.max))`)
+        - Constructor params:
+            - [ ] ilk is named `LITE-PSM-{TOKEN_SYMBOL}-A` (i.e. `LITE-PSM-USDC-A`)
+            - [ ] `gem` matches the expected token address
+            - [ ] `daiJoin` matches `MCD_JOIN_DAI` from the chainlog
+            - [ ] `pocket` matches the pocket address in the Exec Sheet
+    - `DssLitePsmMom`
+        - [ ] Uses the agreed best Solidity compiler version at the time
+        - [ ] Optimizer is **enabled**
+        - [ ] Contract has been reviewed by at least a couple of DAO dev teams
+        - [ ] Contract has been audited by an external party
+            - [ ] Audit reports are public
+            - [ ] All issues found were fixed or acknowledged
+            - [ ] No code changes are done after final Audit report
+        - [ ] Deployed contract matches the code in the [repo](https://github.com/makerdao/dss-lite-psm/)
+        - [ ] Deployer is no longer the `owner`
+        - [ ] `MCD_PAUSE_PROXY` is the `owner` of the contract (i.e. `require(MomLike(MOM).owner() == MCD_PAUSE_PROXY`))
+    - `LitePsmJob`
+        - [ ] Uses the default Solidity version in [`dss-cron`](https://github.com/makerdao/dss-cron)
+        - [ ] Optimizer is **enabled**
+        - [ ] Deployed contract matches the code in the [repo](https://github.com/makerdao/dss-cron)
+        - Constructor params:
+            - [ ] `_sequencer` matches `CRON_SEQUENCER` address from the chainlog
+            - [ ] `_litePsm` matches `DssLitePsm` address
+            - [ ] `_rushThreshold` matches the Exec Sheet (it might be named as "fill threshold")
+            - [ ] `_gushThreshold` matches the Exec Sheet (it might be named as "trim threshold")
+            - [ ] `_cutThreshold` matches the Exec Sheet (it might be named as "chug threshold")
+- Initialization
+    - `DssLitePsm`:
+        - [ ] `vow` is set to `MCD_VOW` from the chainlog
+        - [ ] `MCD_PAUSE_PROXY` is whitelisted to execute swaps with no fees (i.e. `KissLike(LITE_PSM).kiss(MCD_PAUSE_PROXY)`)
+        - [ ] `DssLitePsmMom` is authed (i.e. `RelyLike(LITE_PSM).rely(MOM)`)
+    - `DssLitePsmMom`:
+        - [ ] The chief (`MCD_ADM`) is set as the `authority` (i.e. `MomLike(MOM).setAuthority(MCD_ADM)`)
+    - `LitePsmJob`:
+        - [ ] Job is added to `CRON_SEQUENCER`
```

Comment:
Don't we need to match the source of the contract?

Reply:
That's a good question. I don't think this was ever validated before, but we can add it for completeness.

Reply:
I think source code check was not added yet

Reply:
It is: https://github.com/makerdao/pe-checklists/blob/12eb378be1fc7b2ede88092b58f4d41294fe3f02/spell/psm-checklists.md?plain=1#L65